doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,165 @@
+SciPy Code of Conduct
+=====================
+
+
+Introduction
+------------
+
+This code of conduct applies to all spaces managed by the SciPy project,
+including all public and private mailing lists, issue trackers, wikis, blogs,
+Twitter, and any other communication channel used by our community.  The SciPy
+project does not organise in-person events, however events related to our
+community should have a code of conduct similar in spirit to this one.
+
+This code of conduct should be honored by everyone who participates in
+the SciPy community formally or informally, or claims any affiliation with the
+project, in any project-related activities and especially when representing the
+project, in any role.
+
+This code is not exhaustive or complete. It serves to distill our common
+understanding of a collaborative, shared environment and goals. Please try to
+follow this code in spirit as much as in letter, to create a friendly and
+productive environment that enriches the surrounding community.
+
+
+Specific Guidelines
+-------------------
+
+We strive to:
+
+1. Be open. We invite anyone to participate in our community. We prefer to use
+   public methods of communication for project-related messages, unless
+   discussing something sensitive. This applies to messages for help or
+   project-related support, too; not only is a public support request much more
+   likely to result in an answer to a question, it also ensures that any
+   inadvertent mistakes in answering are more easily detected and corrected.
+
+2. Be empathetic, welcoming, friendly, and patient. We work together to resolve
+   conflict, and assume good intentions. We may all experience some frustration
+   from time to time, but we do not allow frustration to turn into a personal
+   attack. A community where people feel uncomfortable or threatened is not a
+   productive one.
+
+3. Be collaborative. Our work will be used by other people, and in turn we will
+   depend on the work of others. When we make something for the benefit of the
+   project, we are willing to explain to others how it works, so that they can
+   build on the work to make it even better. Any decision we make will affect
+   users and colleagues, and we take those consequences seriously when making
+   decisions.
+
+4. Be inquisitive. Nobody knows everything! Asking questions early avoids many
+   problems later, so we encourage questions, although we may direct them to
+   the appropriate forum. We will try hard to be responsive and helpful.
+
+5. Be careful in the words that we choose.  We are careful and respectful in
+   our communication and we take responsibility for our own speech. Be kind to
+   others. Do not insult or put down other participants.  We will not accept
+   harassment or other exclusionary behaviour. This includes, but is not
```

Comment:
I humbly beg you to remove 'but is not limited to' - it is legal boilerplate.   "Examples are:" has just the same meaning, and is much easier to read.

Reply:
Just "Examples are" is less clear as there's no reference to the previous sentence. I much prefer the current version, and that's not legal jargon but plain English.

Reply:
No, 'includes, but is not limited to' is [characteristic of legal documents](http://www.lawprose.org/lawprose-lesson-226-including-but-not-limited-to/), and is [terrible style](https://english.stackexchange.com/questions/4290/how-to-simplify-this-can-include-but-not-limited-to).  I'm not wedded to my rephrasing, but any rephrasing that doesn't use that particular phrase would make me a lot happier.    I can't see how "Examples are" does less of a job of referring to the previous sentence than "This includes" but how about "Examples of such behavior are:"?

Reply:
How about:

> We will not accept harassment or other exclusionary behaviors, such as:

Reply:
Yes, that's fine by me.